Sources/SwiftSyntaxBuilder/gyb_generated/BuildableNodes.swift

```diff
@@ -9052,15 +8842,15 @@ public struct WhileStmt: StmtBuildable, ExpressibleAsWhileStmt {
     labelName: String?,
     labelColon: TokenSyntax? = nil,
     whileKeyword: TokenSyntax = TokenSyntax.`while`,
-    body: ExpressibleAsCodeBlock,
-    @ConditionElementListBuilder conditionsBuilder: () -> ExpressibleAsConditionElementList = { ConditionElementList([]) }
```

Comment:
Hm, `ConditionElement` was actually a node I intentionally excluded since it introduces a second builder closure parameter, see https://github.com/apple/swift-syntax/pull/465#discussion_r901814595

Reply:
❌ I agree that this should not be a result builder because making the conditions because the body is without doubt the primary body of the node and making the conditions a result builder significantly reduces legibility IMHO. Also, IMO, the usual case is that `if` and `guard` statements have a single condition so it won’t benefit much from result builders.